Homework_2.sql

```diff
@@ -0,0 +1,33 @@
+USE baseball;
+SHOW TABLES;
+
+#calculation of annual average
+DROP TABLE IF EXISTS Annual_Average;
+CREATE TABLE Annual_Average as
+select b.batter, YEAR(g.local_date),
+    sum(Hit)/nullif(sum(atBat),0) as annual_avg
+    FROM game g JOIN batter_counts b on g.game_id = b.game_id group by b.batter, year(g.local_date)
+    order by  1,2;
```

Comment:
I liked the way date is casted to year wrapper.

Minor: The order by statement could be modified to specify the column names, though it doesn't make any functional change in this query.

Reply:
modified this change.